crates/polars-stream/src/nodes/io_sources/parquet/init.rs

```diff
@@ -71,9 +69,8 @@ impl ParquetSourceNode {
 
         let use_statistics = self.options.use_statistics;
 
-        let (mut raw_morsel_senders, raw_morsel_receivers): (Vec<_>, Vec<_>) =
-            (0..self.config.num_pipelines).map(|_| connector()).unzip();
-
+        let (mut raw_morsel_sender, raw_morsel_receivers) =
+            distributor_channel(self.config.num_pipelines, DEFAULT_DISTRIBUTOR_BUFFER_SIZE);
```

Comment:
I think we should use a channel capacity of 1 here - the receiver of this (down below) basically immediately consumes and then re-sends the morsel, then there is channel capacity again.

Reply:
That holds in most cases, but because we're crossing threads here I'd prefer it if there to be some slack.